src/pages/Community/FeedList/FeedList.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import result from "./POSTDATA";
+import ItemWrap from "./ItemWrap.js";
+import "./FeedList.scss";
+
+// const API = "http://10.58.7.154:8000/post";
+
+class FeedList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      data: result,
+    });
+  }
```

Comment:
- data보다 좋은 변수명이 있을 것 같습니다. 해당 변수에 담긴 내용을 더욱 잘 설명하는 이름을 지어주세요
- CDM에서 한번 더 setState 할 필요 없이 처음부터 constuctor에 초기값을 부여하면 될 것 같습니다
```suggestion
class FeedList extends Component {
  constructor() {
    super();
    this.state = {
      data: result,
    };
  }
```

Reply:
수정했습니다.